operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1298,90 +1323,90 @@ spec:
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=entity-operator
                         rht.subcomp_t=infrastructure
                     - name: STRIMZI_CUSTOM_KAFKA_MIRROR_MAKER2_LABELS
                       value: |
                         com.company=Red_Hat
                         rht.prod_name=Red_Hat_Application_Foundations
-                        rht.prod_ver=2024.Q2
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=kafka-mirror-maker2
                         rht.subcomp_t=application
                     - name: STRIMZI_CUSTOM_KAFKA_MIRROR_MAKER_LABELS
                       value: |
                         com.company=Red_Hat
                         rht.prod_name=Red_Hat_Application_Foundations
-                        rht.prod_ver=2024.Q2
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=kafka-mirror-maker
                         rht.subcomp_t=application
                     - name: STRIMZI_CUSTOM_CRUISE_CONTROL_LABELS
                       value: |
                         com.company=Red_Hat
                         rht.prod_name=Red_Hat_Application_Foundations
-                        rht.prod_ver=2024.Q2
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=cruise-control
                         rht.subcomp_t=application
                     - name: STRIMZI_CUSTOM_KAFKA_BRIDGE_LABELS
                       value: |
                         com.company=Red_Hat
                         rht.prod_name=Red_Hat_Application_Foundations
-                        rht.prod_ver=2024.Q2
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=kafka-bridge
                         rht.subcomp_t=application
                     - name: STRIMZI_CUSTOM_KAFKA_EXPORTER_LABELS
                       value: |
                         com.company=Red_Hat
                         rht.prod_name=Red_Hat_Application_Foundations
-                        rht.prod_ver=2024.Q2
+                        rht.prod_ver=2024.Q4
                         rht.comp=AMQ_Streams
-                        rht.comp_ver=2.7
+                        rht.comp_ver=2.8
                         rht.subcomp=kafka-exporter
                         rht.subcomp_t=application
```

Comment:
The repetition here is horrible but thats wrangling for another day.

Reply:
Put the common fields in another env and referenced it in the other envs, take a look and let me know what you think